genrify/datasources/datasources.py

```diff
@@ -0,0 +1,7 @@
+import opendatasets as od
+import shutil
+
+def download_datasets():
+    # download gtzan
+    od.download_kaggle_dataset("https://www.kaggle.com/datasets/andradaolteanu/gtzan-dataset-music-genre-classification", "./")
+    shutil.rename('gtzan-dataset-music-genre-classification', 'Data')
```

Comment:
The file structure here is 
gtzan-dataset-music-genre-classification - 
    |---- Data
    |      |---- genre1
    etc.

So we can't just rename the data. Please have a look at what I currently done

Reply:
The issue should be resolved now. We are now downloading from osf which doesn't require a kaggle account

Reply:
oh I see. Awesome